graylog2-web-interface/src/views/components/contexts/SearchPageLayoutProvider.jsx

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+import * as React from 'react';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import SearchPageLayoutContext from './SearchPageLayoutContext';
+
+const defaultLayoutConfig = {
+  sidebar: {
+    isInline: false,
```

Comment:
Could we load the default from localStorage or user preferences?

Reply:
Yes, the plan is to save this information as a user preference in another iteration.